### ISSUE TITLE
I suggest adding：Custom subnet  and ip address ?

### ISSUE THREAD

**Issue by szzylph** [netbirdio/netbird#1617]
I suggest adding：
Custom subnet
Specify the host IP address (specify the static IP address and multiple IP addresses)

**Comment by longng78** [comment]
yes this would be very hopeful and useful. some of us have want to host service that require https so being able to set static ip for those system so we can point dns properly and https match up

**Comment by runmark** [comment]
+1

**Comment by jary888** [comment]
+1

**Comment by t3cneo** [comment]
Ability to assign specific /16 or /24 subnet to groups would be great while tickling with the ability to manually setting IPs.

Because Access Control is group based, that would make Netbird a killer SDN

**Comment by flokli** [comment]
Let's not forget subnets for V6 while at it ;-)

**Comment by longng78** [comment]
whats the chance of this being added? would like to switch from tailscale but can't. try looking at the code since I dabble a little bit in coding but don't think I'm looking at the right spot. lol or if someone can point me to the gui portion of the code

**Comment by fdisamuel** [comment]
Being able to set custom IP subnet for all my clients would be amazing! Is there any possibility for this functionality to be implemented in the future? Regards,

**Comment by libDarkstreet** [comment]
I also think it would be a very good feature!

**Comment by mnxvs** [comment]
We need this thing on self-hosted too!

**Comment by gtrott808** [comment]
https://github.com/netbirdio/netbird/issues/1633#issuecomment-2106228602

work around is here but, how is this not configurable?? seems odd this isn't a normal feature. I can only assume on the SaaS version you can change this? If not, that's a deal breaker for sure. We have specific subnets, VRFs, policies, groups etc that using a /10 is comical.

**Comment by PapaZigE** [comment]
+1 For Self-hosted. This along with an integrated custom Name Server is the only thing lacking with the switch from Netmaker. I'm really liking Netbird!

**Comment by W1zzardTPU** [comment]
Wanted to switch from Netmaker/Zerotier, found out that IP ranges can't be controlled, giving up on Netbird, account deleted. 

Why isn't this a feature?

**Comment by simpsonnth** [comment]
+1 self hosted

**Comment by dontshootitsme** [comment]
+1
The netbird DNS service is not working for me. Because I have to point nameserver to 127.0.0.1( mosdns forwarding to DoT servers ).

**Comment by dontshootitsme** [comment]
I upgraded the self-hosted netbird server to the newest version (0.56.1) and I can edit the peers' IP addresses now.

<img width="1205" height="481" alt="Image" src="https://github.com/user-attachments/assets/59b9d88c-b251-4e26-98ce-67708457b5b5" />

Good job !
